bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -348,7 +350,10 @@ Job create(JobInfo jobInfo) {
 
   public TableResult query(String sql) {
     try {
-      return bigQuery.query(QueryJobConfiguration.of(sql));
```

Comment:
please convert to use `JobConfigurationFactory.createQueryJobConfigurationBuilder()`

Reply:
Ack.